stginga/plugins/local_plugin_mixin.py

```diff
@@ -18,28 +19,21 @@
     pass
 
 # STGINGA
+import stginga
 from stginga import utils
 
 __all__ = ['HelpMixin', 'MEFMixin', 'ParamMixin']
 
 
+# base of our online documentation
+rtd_base_url = "https://stginga.readthedocs.io/en/"
+
+
 class HelpMixin(object):
     def help(self):
         """Display online help for the plugin."""
-        if not self.fv.gpmon.has_plugin('WBrowser'):
```

Comment:
Maybe instead of deleting this whole block, we need to do a Ginga version check and still call this for Ginga 4.x ? Or is there a better way? Hope you can advise. Thanks!